checks/remotesettings/utils.py

```diff
@@ -69,6 +70,13 @@ async def get_records(self, *args, **kwargs) -> List[Dict]:
             None, lambda: self._client.get_records(*args, **kwargs)
         )
 
+    @retry_timeout
+    async def get_monitor_changes(self, bust_cache=False, **kwargs) -> List[Dict]:
+        if bust_cache:
+            random_cache_bust = random.randint(999999000000, 999999999999)
+            kwargs.setdefault("_expected", random_cache_bust)
```

Comment:
Another option would be to error out if `_expected` already is in `kwargs`, because in that case the intention is rather unclear.